App.js

```diff
@@ -1,39 +1,84 @@
 import React, { useState, useCallback } from 'react';
-import { StyleSheet, Text, View, FlatList, Dimensions, TouchableHighlight, Button } from 'react-native';
+import {
+  StyleSheet,
+  Text,
+  View,
+  FlatList,
+  Dimensions,
+  TouchableHighlight,
+  Button,
+  Image,
+} from 'react-native';
 import { newBoard } from './newboard.js';
 
 const numColumns = 7;
+const newTurnTotals = [0, 0];
 
 export default function App() {
   const [
-    { roll, even, odd, position1, message, optMessage, score, board, endOfGame, about },
+    {
+      roll,
+      even,
+      odd,
+      position1,
+      position2,
+      message,
+      optMessage,
+      score,
+      board,
+      endOfGame,
+      about,
+      numberOfPlayers,
+      whoseTurn,
+      turnTotals,
+    },
     setGameState,
   ] = useState({
     roll: 0,
     even: false,
     odd: false,
     position1: 0,
-    message: 'Roll to Start',
-    optMessage: 'Player',
+    position2: 0,
+    message: '1-Player Game',
+    optMessage: '2-Player Game',
     score: 0,
     board: JSON.parse(JSON.stringify(newBoard)),
     endOfGame: false,
     about: true,
+    numberOfPlayers: 0,
+    whoseTurn: 0,
+    turnTotals: JSON.parse(JSON.stringify(newTurnTotals)),
   });
 
   // render board
   const renderBoard = ({ item }) => {
```

Comment:
Is this actually renderBoardItem?

Reply:
Yes...it is a function that scrolls through all of the board items.